src/@ractf/api/challenge.js

```diff
@@ -79,6 +79,10 @@ export const editChallenge = ({
     });
 };
 
+export const incrementSolveCount = (id, blood_name) => {
```

Comment:
`blood_name` isn't an especially good name for this variable (reflected elsewhere too) as "blood" is used fairly consistently as a contraction of "first blood", rather than as a generic term for a solve. We only know if it's a first blood or not when assigning to the `first_blood_name` attribute on challenges. At this point it's just the username of the team who performed this solve, and we have no way to know otherwise.